bib/cmd/bootc-image-builder/main.go

```diff
@@ -407,7 +407,7 @@ func run() error {
 	manifestCmd.Flags().String("config", "", "build config file")
 	manifestCmd.Flags().String("rpmmd", "/rpmmd", "rpm metadata cache directory")
 	manifestCmd.Flags().String("target-arch", "", "build for the given target architecture (experimental)")
-	manifestCmd.Flags().StringArray("type", []string{"qcow2"}, "image types to build [qcow2, ami, iso, raw]")
+	manifestCmd.Flags().StringArray("type", []string{"qcow2"}, "image types to build [qcow2, ami, iso, raw, vmdk, anaconda-iso]")
```

Comment:
This one makes total sense, but I think https://github.com/osbuild/bootc-image-builder/pull/269 will get us to a slightly more sustainable place.  That said now looking at it I realize I should also update the above for the Go error place too.

Reply:
@cgwalters i can incorporate your implementation if that helps

Reply:
I'm not sure we want the changes to README.md; do you agree with my comments?

Reply:
Sure, feel free to pull in my PR changes into this!

Reply:
@cgwalters done